modules/webhooks.py

```diff
@@ -24,7 +29,30 @@ def __init__(self, config, system_webhooks, notifiarr=None, apprise=None):
         self.notifiarr = notifiarr
         self.apprise = apprise
 
+    def request_and_check(self, webhook_type, json):
+        """
+        Send a webhook request and check for errors.
+        retry up to 6 times if the response is a 500+ error.
+        """
+        retry_count = 0
+        retry_attempts = 6
+        request_delay = 2
+        for retry_count in range(retry_attempts):
+            if webhook_type == "notifiarr":
+                response = self.notifiarr.notification(json=json)
+            else:
+                if webhook_type == "apprise":
+                    json["urls"] = self.apprise.notify_url
+                    webhook_type = f"{self.apprise.api_url}/notify"
+            response = self.config.post(webhook_type, json=json)
+            if response.status_code < 500:
+                return response
+            time.sleep(request_delay)
+            retry_count += 1
+        raise Failed(f"({response.status_code} [{response.reason}]) after {retry_attempts} attempts.")
```

Comment:
We probably don't want to raise a Failed here but just use `logger.warning` as webhook notifications aren't essential and it would be a bad user experience to crash qbit_manage just because of notification failures.